backend/src/main/java/com/roc/app/user/referee/licence/RefereeLicenceController.java

```diff
@@ -0,0 +1,50 @@
+package com.roc.app.user.referee.licence;
+
+import com.roc.app.user.referee.exception.RefereeNotFoundException;
+import com.roc.app.user.referee.general.dto.RefereeResponseDto;
+import com.roc.app.user.referee.licence.dto.RefereeAddLicenceRequestDto;
+import com.roc.app.user.referee.licence.dto.RefereeAddLicenceResponseDto;
+import com.roc.app.user.referee.licence.dto.RefereeLicenceResponseDto;
+import jakarta.persistence.EntityNotFoundException;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/licence")
```

Comment:
Should be plural form:
```suggestion
@RequestMapping("/api/licences")
```